src/common/styles/mixins.sass

```diff
@@ -96,8 +161,18 @@
 
 @mixin body-text-3-opensans
   font-family: "Open Sans", sans-serif
-  font-size: 0.813rem !important
-  line-height: 150% !important
+  font-size: 0.75rem !important
+  line-height: 1rem !important
+  font-style: normal
+  font-weight: normal
```

Comment:
remove all font weight, because CSS already set this default to "normal"

Reply:
removed.